File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/UnityCommands/delete_player_pref.py
from altunityrunner.commands.base_command import BaseCommand

class DeletePlayerPref(BaseCommand):
    def __init__(self, socket,request_separator,request_end):
        super().__init__(socket,request_separator,request_end)
    
    def execute(self):
        print('Delete all Player Prefs')
        data = self.send_data(self.create_command('deletePlayerPref'))
        return self.handle_errors(data)

File: Assets/AltUnityTester/AltUnityDriver/Commands/UnityCommands/DeletePlayerPref.cs
public class DeletePlayerPrefDriver : AltBaseCommand
{
    public DeletePlayerPrefDriver(SocketSettings socketSettings) : base(socketSettings)
    {
    }
    public void Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("deletePlayerPref")));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}